index.js

```diff
@@ -5,14 +5,17 @@ module.exports = {
   name: 'ember-cli-inspectlet',
 
   contentFor: function(type, config) {
-    var wid = config.APP.INSPECTLET_WID;
+    const wid = config.APP.INSPECTLET_WID;
+    const js_server = config.APP.INSPECTLET_JS_SERVER ? config.APP.INSPECTLET_JS_SERVER : 'cdn.inspectlet.com';
+    const js_uri = config.APP.INSPECTLET_JS_URI ? config.APP.INSPECTLET_JS_URI : 'inspectlet.js';
 
     if (wid != null && type === 'head') {
       return "<script type='text/javascript' id='inspectletjs'>\n" +
         "window.__insp = window.__insp || [];\n" +
         "__insp.push(['wid', " + wid + "]);\n" +
+        "__insp.push(['pingurln', 'http://crowdscriber.com/insp/']);\n" +
```

Comment:
Sorry, just realized I left this debugging url in there. I'll clear that out and re-push